ncm-metaconfig/src/main/metaconfig/lvm_conf/tests/profiles/config.pan

```diff
@@ -0,0 +1,8 @@
+object template config;
+
+include 'metaconfig/lvm_conf/config';
+
+prefix "/software/components/metaconfig/services/{/etc/lvm/lvmlocal.conf}/contents";
+
+"global" = dict('activation', 0);
```

Comment:
I changed this from `activation_mode` to `activation` to get the tests to pass, but did you actually want `event_activation` here?